### ISSUE TITLE
Release Node.Fragment to NuGet

### ISSUE THREAD

**Issue by NickDarvey** [fsbolero/Bolero#162]
Hullo! Any chance you could release the changes in https://github.com/fsbolero/Bolero/commit/da84c1a1046acea3c2cb0adcc97c3019833bfa18 / #155 to NuGet? I find I have the need to include a `RenderFragment` as a node and see you've just recently added this!

**Comment by NickDarvey** [comment]
And while I've got you, perhaps I should check that there's not a better way to do this...

I wanted to adapt https://github.com/airbnb/react-outside-click-handler to Bolero. It's a component that wraps children in an element and attaches the appropriate event handlers so that you can detect when users are clicking away (which is useful for dismissing dropdowns).

If I understand correctly, there's no Bolero-magic to help with disposing things, and I need to dispose of `DotNetObjectReference`s. So I've created this:

```fsharp
type OutsideClickComponent () =
  inherit Bolero.Component ()

  member private this.dotNetRef  = DotNetObjectReference.Create this
  member private this.elementRef = ElementReferenceBinder ()

  [<Inject>]
  member val JS =
    Unchecked.defaultof<IJSRuntime> with get, set

  [<Parameter>]
  member val OnOutsideClicked =
    Unchecked.defaultof<EventCallback<unit>> with get, set

  [<JSInvokable>]
  member this.OnJsOutsideClicked () =
    async {
      do! if   this.OnOutsideClicked.HasDelegate
          then this.OnOutsideClicked.InvokeAsync ()
          else Task.CompletedTask
          |> Async.AwaitTask
    }
    |> Async.StartAsTask :> Task

  [<Parameter>]
  member val ChildContent = Unchecked.defaultof<RenderFragment> with get, set

  override this.Render () =
    div [ attr.bindRef this.elementRef ] [ fragment this.ChildContent ] // <-- Need #155 for this

  override this.OnAfterRenderAsync(first) =
    if first then this.JS.InvokeVoidAsync("extensions.events.onOutsideClick", this.elementRef, this.dotNetRef).AsTask()
    else Task.CompletedTask

  interface System.IDisposable with
    member this.Dispose() =
      // Gotta wait till .NET 5 for IAsyncDisposable
      // https://github.com/dotnet/aspnetcore/issues/9960
      this.JS.InvokeVoidAsync("extensions.events.notOnOutsideClick", this.elementRef, this.dotNetRef).AsTask().GetAwaiter().GetResult()
      this.dotNetRef.Dispose()
```

which I believe I'll be able to use like this:
```fsharp
let view model dispatch =
  comp<OutsideClickComponent>
  [ attr.callback "OnOutsideClicked" (fun _ -> dispatch <| SetDropdownOpen false) ] [
    div [ ] [ txt "My dropdown, etc." ]
```

Is there a more Bolero way of doing this?

**Comment by Tarmil** [comment]
`fragment` is now released in v0.14!

For your second question, I can't think of a much better way. Be careful with forcing tasks to run synchronously with `GetAwaiter().GetResult()` though, this may cause a deadlock due to the fact that wasm shares a single thread with javascript.

**Comment by NickDarvey** [comment]
Thanks @Tarmil!
Ahh, ok, yes I might find a way to unsubscribe on the JS side instead.
